templates/widget/form/tlmswidgetform.php

```diff
@@ -1,8 +1,8 @@
 <p>
-	<label for="<?php echo esc_attr($titleId); ?>"><?php esc_attr_e('Title:', 'talentlms'); ?></label>
-	<input class="widefat"
-		   id="<?php echo esc_attr($titleId); ?>"
-		   name="<?php echo esc_attr($titleName); ?>"
-		   type="text"
-		   value="<?php echo esc_attr($title); ?>">
+    <label for="<?php echo esc_attr($titleId); ?>"><?php esc_attr_e('Title:', 'talentlms'); ?></label>
+    <input class="widefat"
+           id="<?php echo esc_attr($titleId); ?>"
+           name="<?php echo esc_attr($titleName); ?>"
+           type="text"
+           value="<?php echo $title; ?>">
```

Comment:
Why we have removed esc_attr() from title variable?

Reply:
because it is into a value element